mrmustard/math/__init__.py

```diff
@@ -34,15 +34,19 @@
 
     math.cos(x)  # torch backend
 """
-
-
 import importlib
+from julia.api import Julia
 from mrmustard import settings
 
 if importlib.util.find_spec("tensorflow"):
-    from mrmustard.math.tensorflow import TFMath
+    from mrmustard.math.tensorflow_wrapper import TFMath
 if importlib.util.find_spec("torch"):
-    from mrmustard.math.torch import TorchMath
+    from mrmustard.math.torch_wrapper import TorchMath
+
+# initialize Julia
+jl = Julia(
```

Comment:
Are this and `from julia import Main as Main_julia` fast, or do they contribute significantly to the import time?

Reply:
It seems that `jl = Julia(compiled_modules=False)` is rather slow. Should I for example make a settings entry `init_julia` and only run this line when `init_julia == True`?

![image](https://github.com/XanaduAI/MrMustard/assets/52749580/1c4dfe41-cd0a-4662-9b85-e677a6ad4ac2)

Reply:
Or you could import it in the setter for the setting that you added..?
`If precision is ..., import julia`